examples/flutter_gallery/pubspec.yaml

```diff
@@ -7,6 +7,7 @@ dependencies:
   string_scanner: 1.0.2
   url_launcher: 0.4.2+5
   cupertino_icons: 0.1.1
+  video_player: ^0.0.2
```

Comment:
I think Flutter has a convention of pinning dependencies, i.e. `video_player: 0.0.2`

Reply:
Done